src/app/public/modules/ag-grid/ag-grid.service.spec.ts

```diff
@@ -266,6 +271,58 @@ describe('SkyAgGridService', () => {
     });
   });
 
+  describe('onKeyPress', () => {
+    let onKeyPressFunction: Function;
+    let gridApi: GridApi;
+    let keypress: CellKeyPressEvent;
+    let event: KeyboardEvent;
```

Comment:
Does this need to be declared here? It looks like it's only initialized and used in each individual test rather than initialized in `beforeEach()` like the other variables declared outside the test functions.